projects.go

```diff
@@ -62,6 +62,7 @@ type Project struct {
 	SnippetsEnabled                           bool               `json:"snippets_enabled"`
 	ResolveOutdatedDiffDiscussions            bool               `json:"resolve_outdated_diff_discussions"`
 	ContainerRegistryEnabled                  bool               `json:"container_registry_enabled"`
+	PackagesEnabled                           bool               `json:"packages_enabled"`
```

Comment:
Please place this one between `MirrorOverwritesDivergedBranches` and `ServiceDeskEnabled` so it follows the ordering in the docs. Thanks!

Reply:
Done in the commit below.